azure-pipelines.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           # -timeout 25m as our tests typically run between 10-15m and I'd rather have some headroom.
           # "-check.v" (must be after package list) outputs timings
           set -e
-          go test -timeout 25m -race -short -cover ./cmd ./common ./common/parallel ./ste ./azbfs ./sddl "-check.v"
+          go test -timeout 35m -race -short -cover ./cmd ./common ./common/parallel ./ste ./azbfs ./sddl "-check.v"
```

Comment:
>35m [](start = 27, length = 3)

What prompted the increase in test run time?

Reply:
@nakulkar-msft I know this in bits and pieces. Can you answer this?